### ISSUE TITLE
firebas-element accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/firebase-element#11]
Audit results here: 
[Severe] Controls and media elements should have labels (2)[Warning] Text elements should have a reasonable contrast ratio (4)Not applicable tests (9)Passing tests (4)

One thing I'll mention is that the focus state is really difficult to see on the buttons. The blue focus state on the link and text field is fine, but I'm not able to see it at all on the buttons -- is it actually even going there?
